api/common/authentication/controller.js

```diff
@@ -0,0 +1,39 @@
+import express from 'express';
+const loginRouter = express.Router();
+import Joi from 'joi';
+import login from './authenticate.js';
+import verify from './authenticateToken.js';
+
+const credentialSchema = Joi.object({
+    login: Joi.string().required(),
+    password: Joi.string().required()
+});
+
+loginRouter.post('/', (req, res) => {
```

Comment:
Remove router from the controller. 
The controller should only have a handler for requests and it should not contain a route to this handler.